src/PooledArrays.jl

```diff
@@ -4,6 +4,8 @@ import DataAPI
 
 export PooledArray, PooledVector, PooledMatrix
 
+# TODO: implement compresspool! and compresspool functions that compresses pool of PooledArray
```

Comment:
I'd rather file an issue for this kind of thing.

Reply:
I thought of implementing it in this PR, but you are right, that it is better to keep it separate. See https://github.com/JuliaData/PooledArrays.jl/issues/58